hooks/useCreateOrder.ts

```diff
@@ -16,14 +16,14 @@ export const useCreateOrder = () => {
     const { maker, isCollectionApproved, isTransferManagerApproved } =
       await hypercertExchangeClient.createFractionalSaleMakerAsk({
         startTime: Math.floor(Date.now() / 1000), // Use it to create an order that will be valid in the future (Optional, Default to now)
-        endTime: Math.floor(Date.now() / 1000) + 365 * 24 * 60 * 60, // If you use a timestamp in ms, the function will revert
-        price: parseEther("1"), // Be careful to use a price in wei, this example is for 1 ETH
+        endTime: Math.floor(Date.now() / 1000) + (100 * 365 * 24 * 60 * 60), // If you use a timestamp in ms, the function will revert
+        price: parseEther("0.000001"), // Be careful to use a price in wei, this example is for 1 ETH
         itemIds: [tokenId], // Token id of the NFT(s) you want to sell, add several ids to create a bundle
-        minUnitAmount: 10, // Minimum amount of units to sell per sale
-        maxUnitAmount: 1000, // Maximum amount of units to sell per sale
+        minUnitAmount: 1, // Minimum amount of units to sell per sale
+        maxUnitAmount: parseEther("1"), // Maximum amount of units to sell per sale
```

Comment:
Conflicts with the change you made to only mint 10000 units

Reply:
Not sure which conflict do you mean. Prior 17 claims has minted with this value